src/components/Footer.module.css

```diff
@@ -113,22 +111,48 @@
   display: flex;
   justify-content: space-evenly;
   align-items: center;
-  width: 300px;
+  flex: 2;
 }
 
-.iconContainer > span {
+.iconContainer a {
   display: flex;
-  justify-content: space-around;
+  justify-content: space-between;
 }
 
 .topFooterContainer {
   background-color: rgb(0, 62, 82);
   display: flex;
   justify-content: space-between;
-  padding: 15px 105px;
+  padding: 15px 2%;
```

Comment:
percentages as padding can be weird at really small and really large widths.  I feel like rems/ems or even px is more standard for padding and margin.  I haven't seen the actual specific case I'm just talking generally